docs/source/user/talks.rst

```diff
@@ -0,0 +1,30 @@
+
+Talks and Tutorials
+===================
+
+Note: This is a selection of talks and tutorials have been given by members of
```

Comment:
Should that be a `.. note::` to get the note markup working or is it not that sort of note?